### DISCUSSION TITLE
How much RAM does jaeger needs running in a on prem openshift kubernetes cluster?

### DISCUSSION THREAD

**Discussion by sanasz91mdev** [jaegertracing/jaeger-operator#2125] [Q&A]
Any idea how much RAM would Jaeger agent, collector, query need to run in a production on prem openshift kubernetes cluster of 3 worker, master nodes? 

I did not see any prerequistes in openshift documentation for openshift distributed tracing.

Is there any recommendation in general regardless of any platform openshift/AWS/AKS etc?

**Comment by frzifus** [comment]
Hi @sanasz91mdev, i think it highly depends on the amount of agents that are deployed, queries are made, traces are generated, probably your sampling strategy and other configurations.

@pavolloffay @rubenvp8510 do you know if something like benchmark results or similar exists?
